build/docker/kitchen_terraform/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
 RUN apk add --no-cache \
-        openssh-client=7.7_p1-r3
+        openssh-client
```

Comment:
need concrete version here

Reply:
Why do we need it?

Reply:
I've removed version because previous was unreachable, can change it to openssh-client=7.7_p1-r4 , if it is necessary.